src/configs/base.ts

```diff
@@ -208,6 +208,16 @@ export const config: Config = {
     },
   ],
   onSurface: [
+    {
+      name: 'background',
+      description:
+        'For use as a background color, in components such as TopBar SearchField.',
```

Comment:
This may be too specific. We should probably avoid naming components

Reply:
`onSurfaceBackground` might be confusing as it is a surface on a surface?

Reply:
We discussed this briefly but I think it makes sense. The purpose was for the search bar to match the background color of the page but that didn't quite meet a11y standards. So background on surface, though it may sound a bit odd, I think makes perfect sense. Open to other suggestions though